src/wamp-server-over-fs.ts

```diff
@@ -28,7 +28,10 @@ export default function wampServerOverFs(fs: FileSystem, port = 3000): Promise<W
             url: `ws://127.0.0.1:${port}/`,
         });
 
+        let isConnectionClosed = true;
```

Comment:
btw, theirs docs very strange :) isConnected is property, not a method, anyway it wasnt present in typings